generator/fluentReportsGenerator.js

```diff
@@ -178,6 +180,43 @@ class FluentReportsGenerator {
         this._UIBuilder = UI;
 
         // Tracking Information
+        this._formatterFunctions =  [];
+        //JSON style functions.
+        if(options.report.formatterFunctions){
```

Comment:
Passing `undefined` as the `report` option results in an error. I think undefined is supposed to be supported because of the `blankReport` option.

Reply:
Thanks for catching this -- I sincerely apologize for not doing the tests before accepting this PR.  This is 100% my fault.